src/arrays.js

```diff
@@ -1,61 +1,79 @@
+const array = ['cat', 'dog', 'elephant', 'fox'];
 const getNthElement = (index, array) => {
-  // your code here
+  return array[index % array.length];
 };
 
+
 const arrayToCSVString = array => {
   // your code here
+  return array.toString();
 };
 
 const csvStringToArray = string => {
   // your code here
+  return string.split(',');
 };
 
 const addToArray = (element, array) => {
   // your code here
+  console.log(array.push(element));
```

Comment:
It is good practice to remove console.log() s from your code once you are finished. This is because you usually don't want to display data to users in the console from within your application/programme where possible. It could be sensitive information and it might also create a lot of noise in case there are other important messages or errors in the console.

Reply:
Just writing `array.push(element)` here would be great